python/rikai/mixin.py

```diff
@@ -126,3 +133,11 @@ def open(self, mode="rb") -> BinaryIO:
         if self.is_embedded:
             return BytesIO(self.data)
         return open_uri(self.uri, mode=mode)
+
+
+class Pretrained(ABC):
+    """Mixin for pretrained model"""
+
+    @abstractmethod
+    def load_pretrained_model(self) -> Any:
```

Comment:
Given that this is the only method of pretrained Mixin. We can just call this name `pretrained` or `pretrained_model()`?